nirc_ehr/src/org/labkey/nirc_ehr/table/NIRC_EHRCustomizer.java

```diff
@@ -124,6 +127,20 @@ public void customize(TableInfo table)
             {
                 customizeObservationSchedule(ti);
             }
+
+            if (matches(ti, "study", "pairings") ||
+                    matches(ti, "study", "pairingSummary"))
```

Comment:
Since pairingSummary is derived from pairings, you shouldn't need to customize it again. It should be able to pull through the updatePairing column in the query.